database/migrations/2016_09_15_193019_add_new_settings_and_emails.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+use App\Models\Email;
+use App\Models\Setting;
+use Illuminate\Database\Migrations\Migration;
+
+class AddNewSettingsAndEmails extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // New Settings
+        if (!(Setting::where('key', 'background_image', '=')->first())) {
+            Setting::insert([
+                'category'  => 'general',
+                'key'       => 'background_image',
+                'value'     => '',
+                'type'      => 'image',
+            ]);
+        }
+
+        if (!(Setting::where('key', 'custom_font_kit_id', '=')->first())) {
+            Setting::insert([
+                'category'  => 'appearance',
+                'key'       => 'custom_font_kit_id',
+                'value'     => '',
+                'type'      => 'text',
+            ]);
+        }
+
+        if (!(Setting::where('key', 'custom_font_name', '=')->first())) {
+            Setting::insert([
+                'category'  => 'appearance',
+                'key'       => 'custom_font_name',
+                'value'     => '',
+                'type'      => 'text',
+            ]);
+        }
+
+        // New Emails
+        Email::firstOrNew([
```

Comment:
You can use `Email::firstOrCreate([...]);` here so you don't have to manually `->save();` at the end.